test/HeapProfilerAgent.js

```diff
@@ -188,23 +184,23 @@ describe('HeapProfilerClient', function() {
 });
 
 function initializeProfiler(done) {
-  launcher.runPeriodicConsoleLog(true, function(childProcess, client) {
-    var injectorClient = new InjectorClient({}, client);
-    debuggerClient = client;
-    heapProfilerClient = new HeapProfilerClient({}, debuggerClient);
-    heapProfilerAgent = new HeapProfilerAgent(
-      {},
-      debuggerClient,
-      injectorClient,
-      frontendClient,
-      heapProfilerClient);
+  launcher.runPeriodicConsoleLog(true, function(childProcess, session) {
+    debuggerClient = session.debuggerClient;
+    frontendClient = session.frontendClient;
+
+    var injectorClient = new InjectorClient({}, session);
+    session.injectorClient = injectorClient;
+
+    heapProfilerClient = new HeapProfilerClient({}, session);
+    session.heapProfilerClient = heapProfilerClient;
+
+    heapProfilerAgent = new HeapProfilerAgent({}, session);
```

Comment:
But in this way all our tests will be dependent on all our modules.
If I test `ConsoleAgent`, I don't want to initialize `HeapProfilerAgent`, but this will happens, because I initialise `SessionStub`, what will contain `HeapProfilerAgent` initialization.
Or I'm wrong here?